applications/ContactStructuralMechanicsApplication/symbolic_generation/penalty_frictionless_mortar_condition/penalty_frictionless_mortar_contact_condition.cpp

```diff
@@ -66,6 +66,143 @@ PenaltyMethodFrictionlessMortarContactCondition<TDim,TNumNodes, TNormalVariation
 /***********************************************************************************/
 /***********************************************************************************/
 
+template< SizeType TDim, SizeType TNumNodes, bool TNormalVariation, SizeType TNumNodesMaster >
```

Comment:
why does this file not have syntax-highlighting?

Reply:
github does not recognize this as cpp-file, there is no syntax-highlighting

Reply:
Probably becasue large file

Reply:
hmh could be